### ISSUE TITLE
Travis CI Automated Builds and SCA

### ISSUE THREAD

**Issue by matthoppe** [freeorion/freeorion#611]
It might be nice to have the Build Status Tag showing on the readme.md if a push or merge broke the baseline. Additionally, can we get it connected up to Coverity Scan, https://scan.coverity.com/travis_ci, for static code analysis? It looks like the project is currently using TravisCI to automate documentation https://travis-ci.org/freeorion/freeorion

**Comment by adrianbroher** [comment]
> It might be nice to have the Build Status Tag showing on the readme.md if a push or merge broke the baseline.

It won't because it gives the false impression that there is anything CI like implemented yet.

> Additionally, can we get it connected up to Coverity Scan, https://scan.coverity.com/travis_ci, for static code analysis?

Maybe, I need to read the limitations of the service first.

> It looks like the project is currently using TravisCI to automate documentation https://travis-ci.org/freeorion/freeorion

That's true. Currently it is only used for the automatic code doc generation.

**Comment by matthoppe** [comment]
> It won't because it gives the false impression that there is anything CI like implemented yet.

Perhaps, it would be more useful if tests were run after the build. However, we would need to start backfilling tests.

**Comment by geoffthemedio** [comment]
Is this implemented now?

**Comment by adrianbroher** [comment]
No, it isn't. Unless you consider *compiles without catching fire* a decent CI test.
